pkgs/development/python-modules/generic/default.nix

```diff
@@ -12,28 +12,18 @@
 
 , buildInputs ? []
 
-# pass extra information to the distutils global configuration (think as global setup.cfg)
-, distutilsExtraCfg ? ""
-
 # propagate build dependencies so in case we have A -> B -> C,
 # C can import propagated packages by A
 , propagatedBuildInputs ? []
 
-# passed to "python setup.py install"
-, setupPyInstallFlags ? []
-
 # passed to "python setup.py build"
+# https://github.com/pypa/pip/issues/881
 , setupPyBuildFlags ? []
 
 # enable tests by default
 , doCheck ? true
 
-# List of packages that should be added to the PYTHONPATH
-# environment variable in programs built by this function.  Packages
-# in the standard `propagatedBuildInputs' variable are also added.
-# The difference is that `pythonPath' is not propagated to the user
-# environment.  This is preferrable for programs because it doesn't
-# pollute the user environment.
+# DEPRECATED: use propagatedBuildInputs
```

Comment:
`pythonPath` and `propagatedBuildInputs` serve different roles, however. The goal of `pythonPath` is to register a package as a runtime dependency, but hide its `PYTHONPATH` from being propagated to downstream packages. This is a very desirable thing to have IMO. Why do we not want to support it?